### PR TITLE
Added comments to test_containers_properties.py

### PR DESCRIPTION
**Pull request by jdemon519**
Purpose or Intent
=================

The test verifies data integrity in the Properties table for pods, routes, projects and services

{{pytest: cfme/tests/containers/test_containers_properties.py -v --use-provider container }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 73206544f9b2d024f837ad1a82d08110dce8ceea:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_containers_properties.py [hunk 1]
```diff
@@ -66,6 +71,10 @@ def test_routes_properties_rel(provider, rel):
                           'resource_version'
                           ])
 def test_projects_properties_rel(provider, rel):
+    """ This module verifies data integrity in the Properties table for
+        projects, all cells in the Properties table should contain
+        valid data
+    """
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]:
```python
    """ This module verifies data integrity in the Properties table for
    projects, all cells in the Properties table should contain valid data

    whatever else needs to be described
    """
```
The first line (the short summary) shall be like that.

Comment (jdemon519) [inline cfme/tests/containers/test_containers_properties.py]: @mfalesni thanks, will change it right now. Still getting used to python commenting style:)

**Comment by dajoRH** [comment]
Lint report for commit 1cb3e5b26d2cd9e76748c32447f70174772bc1e3:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d673f76da6f96fb650a40fc76e44079dd9a4c330:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_containers_properties.py [hunk 2]
```diff
@@ -31,7 +31,8 @@
                           ])
 def test_pods_properties_rel(provider, rel):
     """ This module verifies data integrity in the Properties table for:
```
Comment (jaryn) [inline cfme/tests/containers/test_containers_properties.py]: This is not a module but a test

File: cfme/tests/containers/test_containers_properties.py [hunk 3]
```diff
@@ -50,6 +51,10 @@ def test_pods_properties_rel(provider, rel):
                           'host_name'
                           ])
 def test_routes_properties_rel(provider, rel):
+    """ This module verifies data integrity in the Properties table for:
```
Comment (jaryn) [inline cfme/tests/containers/test_containers_properties.py]: not a module

File: cfme/tests/containers/test_containers_properties.py (hunk 2 above)
Comment (jdemon519) [inline cfme/tests/containers/test_containers_properties.py]: @jaryn Of course, I'll change that

**Comment by dajoRH** [comment]
Lint report for commit a3ee74613699cbe648634d154a324558dfc7f557:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by jdemon519** [comment]
@pavelzag @gshefer @jaryn 
Changed the comments, please review.

**Comment by jdemon519** [comment]
@dajoRH please merge

**Comment by dajoRH** [comment]
Response to comment #247332232:

It seems like all the basic requirements were met, I have therefore merged your PR.

**Pull request merged**